README.md

```diff
@@ -171,8 +171,6 @@ For information about the governance of the Node.js project, see
 **Colin Ihrig** &lt;cjihrig@gmail.com&gt; (he/him)
 * [codebytere](https://github.com/codebytere) -
 **Shelley Vohr** &lt;shelley.vohr@gmail.com&gt; (she/her)
-* [danbev](https://github.com/danbev) -
-**Daniel Bevenius** &lt;daniel.bevenius@gmail.com&gt; (he/him)
```

Comment:
Usual practice is to move entries to the `TSC emeriti` section below.

Reply:
Ah great, I'll do that. Thanks